inference/core/workflows/core_steps/analytics/line_counter/v1.py

```diff
@@ -61,12 +61,11 @@ class LineCounterManifest(WorkflowBlockManifest):
     )
 
     line_segment: Union[list, StepOutputSelector(kind=[LIST_OF_VALUES_KIND]), WorkflowParameterSelector(kind=[LIST_OF_VALUES_KIND])] = Field(  # type: ignore
-        description="Lines (one for each batch) in a format [(x1, y1), (x2, y2)];"
-        " direction of line zone is assumed to be that of direction of vector normal to [(x1, y1), (x2, y2)]",
-        examples=["$inputs.zones"],
+        description="Line in the format [[x1, y1], [x2, y2]] consisting of exactly two points. Direction of the line is from [x1, y1] to [x2, y2].",
+        examples=[[[0, 50], [500, 50]], "$inputs.zones"],
```

Comment:
I know the concept of `normal vector` might not be too intuitive, but direction of line is not strictly from `[x1, y1]` to `[x2, y2]` - this suggests object would have to move along the line.

The direction of line is that of normal vector (hence points order is important)

![image](https://github.com/user-attachments/assets/816c5ce2-7c6c-4be3-ba12-1dfccd1d161f)

Lets imagine that green arrow from the above image is our line counter zone. Left point of that arrow is our `(x1, y1)`, right point is `(x2, y2)`, then normal vector is the red arrow and it signifies the direction in which line will count object as `IN`

Maybe there is better way to explain this so we are accurate?